FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,37 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .success(let (data, httpResponse)):
+				completion(FeedImageMapper.map(data: data, from: httpResponse))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+final class FeedImageMapper {
+	private struct RemoteObj: Decodable {
```

Comment:
Suggestion:
Adding `Obj` to a type name doesn't add any value and can be misleading here because `RemoteObj` is actually a struct not a class.
Since `RemoteObj` is private and accessible only from this file it would be enough to just have `Remote` as a name
```suggestion
	private struct Remote: Decodable {
```